addons/spreadsheet/static/src/o_spreadsheet/errors.js

```diff
@@ -17,6 +16,6 @@ export function isLoadingError(valueOrError) {
 
 export class LoadingDataError extends EvaluationError {
     constructor() {
-        super("", LOADING_ERROR);
+        super(LOADING_ERROR, LOADING_ERROR);
```

Comment:
Why change this ? With the error message, all loading cells have a little error red triangle. The point was to avoid this

Reply:
Yeah my bad, that's some remnants of things I tried when Loading error made everything not works. The problem was that LOADING_ERROR message was translated (see the related)